addons/project_issue/migrations/9.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,70 @@
+---Fields in module 'project_issue'---
+project_issue / project.issue            / categ_ids (many2many)         : DEL relation: project.category
+project_issue / project.issue            / tag_ids (many2many)           : NEW relation: project.tags
+# TODO DONE
+
+project_issue / project.issue            / kanban_state (selection)      : now required, default = normal
+# Nothing to do
```

Comment:
You have to call set_defaults to avoid possible empty values in the 8.0 DB: https://github.com/OCA/openupgradelib/blob/master/openupgradelib/openupgrade.py#L463